admin-dev/themes/new-theme/scss/pages/orders/_orders_view.scss

```diff
@@ -407,35 +405,22 @@
     }
 
     .order-actions {
-      #update_order_status_action_form {
-        .select-status {
-          margin-bottom: 0.9375rem;
-        }
-      }
+      justify-content: center;
 
       .select-status {
-        width: 100%;
-        max-width: inherit;
+        max-width: 206px;
 
         .select-status-colored {
           max-width: inherit;
-          /* stylelint-disable-next-line */
-          margin-right: 0 !important;
+          padding-right: 2.5rem;
+          text-overflow: ellipsis;
```

Comment:
Why modify the `admin-dev/themes/new-theme/scss/pages/orders/_orders_view.scss` file ? It should not be used on the product page.

Reply:
wtf, I didn't do any of this :open_mouth:

Reply:
I have no idea why are these changes here :man_shrugging: I'll check them out

Reply:
oh I see, something happened here (screenshot). Although Im still not aware how :man_shrugging:

![Screenshot from 2021-01-26 11-59-31](https://user-images.githubusercontent.com/31609858/105830349-0f540b80-5fce-11eb-8fe0-6eaa7a789554.png)

Reply:
so apparently there was some mistake, so I've reverted those css changes manually